contacts-app/src/components/Button/Button.jsx

```diff
@@ -1,22 +1,32 @@
 import React from "react";
 import styles from "./Button.module.css";
 import classNames from "classnames";
+import PropTypes from "prop-types";
 
 class Button extends React.Component {
   render() {
-    const type = this.props.type;
+    const buttonType = this.props.buttonType;
     const text = this.props.text;
+    const type = this.props.type;
     return (
       <button
+        type={type}
         onClick={event => {
           this.props.handleClick(event);
         }}
```

Comment:
this can also be written as:
```
onClick={this.props.handleClick}
```

Reply:
I am curious why I don't need to pass the `event` as argument here? I thought this is where the event is handled, so the function will actually be executed rather than just pass down the definition of the function, thus I need to pass the `event` here?
Or otherwise, it is some shorthand syntax?